libs/shared/running-tasks/src/lib/running-tasks-types.ts

```diff
@@ -0,0 +1,20 @@
+export enum TaskStatus {
```

Comment:
for other pieces of code like this that are essentially copied from the nx repo, we've added comments in the nx repo to prompt ppl to coordinate with us when making changes there... might be a good idea here as well